prclz/reblock/reblock.py

```diff
@@ -0,0 +1,205 @@
+import argparse
+import os
+import time
+from pathlib import Path
+from typing import List, Tuple, Callable
+import geopandas as gpd
+import igraph
+import numpy as np
+import pandas as pd
+import tqdm
+from shapely.ops import polygonize
+from shapely.wkt import dumps
+from shapely.geometry import (LinearRing, LineString, MultiLineString,
+                              MultiPoint, MultiPolygon, Point, Polygon)
+from logging import basicConfig, debug, info, warning, error
+
+from .planar_graph import PlanarGraph
+
+def block_to_gadm(block: str) -> str:
+    """Just grabs the GADM from a block"""
+    block_rev = block[::-1]
+    idx = block_rev.index("_") + 1
+    return block[0:-idx]
+
+def add_buildings(graph: PlanarGraph, 
+                  bldg_centroids: List[Point],
+                  ) -> PlanarGraph:
+    """Add bldg centroids to planar graph"""
+
+    total_blgds = len(bldg_centroids)
+    for i, bldg_node in enumerate(bldg_centroids):
+        graph.add_node_to_closest_edge(bldg_node, terminal=True)
+
+    if total_blgds > 0:
+        graph._cleanup_linestring_attr()
+    return graph 
+
+def clean_graph(graph: PlanarGraph) -> PlanarGraph:
+    """
+    Some graphs are malformed bc of bad geometries and 
+    we take the largest connected subgraph
+    """
+    is_conn = graph.is_connected()
+    if is_conn:
+        info("Graph is connected...")
```

Comment:
can we add some more info to these logging statements? a bunch of `Graph is connected....` statements in the log aren't super helpful

dropping the graph name would be a good minimal bit of context to add

Reply:
Yeah I just consolidated and removed these -- they were basically just not being used any more so I reduced it to one and deleted the others